benchmarks/graphics_pipeline/main.cpp

```diff
@@ -30,6 +31,50 @@
 
 using namespace ppx;
 
+class Person
+{
+public:
+    enum class MovementDirection
+    {
+        FORWARD,
+        LEFT,
+        RIGHT,
+        BACKWARD
+    };
+
+    Person()
+    {
+        Setup();
+    }
+
+    // Initialize the location of this person.
```

Comment:
nit: Function comments should follow an implicit "This function...", could you update this one and the ones below. So for this one it would be "Initializes..."

Reply:
Done.